### ISSUE TITLE
Application is unregistered due to TOO_MANY_REQUESTS

### ISSUE THREAD

**Issue by DrachenkoAnastasiia** [smartdevicelink/sdl_core#1212]
## Occurrence:
Sometimes

## Preconditions:
1. SDL is built with EXTERNAL_PROPRIETARY flag
2. Define in .INI file 'AppDirectoryQuota' parameter = 1000000
3. Define in 'preloaded_pt' file 'memory_kb' parameter = 200 for 'pre_DataConsent'
4. SDL and HMI are started. First ignition cycle
5. Connect device

## Steps to reproduce:
1. Register new application => 'pre_DataConsent' policies assigned
2. Send 1st PutFile request with file size = 1.1Mb => SDL->app: success = false, resultCode = "OUT_OF_MEMORY" => Ok
3. Send 2nd PutFile request with file size = 0.3Mb => SDL->app: success = true, resultCode = "SUCCESS" = "OUT_OF_MEMORY" => Ok
4. Perform steps 2-3 one more time
--
Or run script [292_ATF_P_Memory_Kb_Constraints_Ignoring.lua](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_scripts/Polices/Validation_of_PolicyTables/292_ATF_P_Memory_Kb_Constraints_Ignoring.lua)

## Actual result:
Application is unregistered due to TOO_MANY_REQUESTS.
Scenario is failed.

## Expected result:
In step 4 requests processed in the same way as in steps 1-2.
Scenario is passed.

## Environment
repo: https://github.com/smartdevicelink/sdl_core/
branch: feature/external_proprietary_policy (head of branch)
ATF version: 2.2, commit 752d9aec810ccb2a30ab6b53c587f8bb8ef1c545

## Expected delivery
- [ ] Source code updates
- [ ] Code comments
- [ ]  UTs add/update
- [ ]  ATF tests add/update
- [ ]  Add/update CI plans/jobs
- [ ]  SDD updates
- [ ]  Guidelines update (sdl_core_guides)
- [ ]  Guidelines update (sdl_hmi_integration_guidelines)

**Comment by JackLivio** [comment]
I was not able to reproduce this issue

Is this how I was supposed to define pre_DataConsent in the preloaded pt?
```
            "pre_DataConsent": {
                "keep_context": false,
                "steal_focus": false,
                "priority": "NONE",
                "default_hmi": "NONE",
                "groups": ["BaseBeforeDataConsent"],
                "memory_kb": 200
            }

```

**Comment by DrachenkoAnastasiia** [comment]
@Itileda, please, take care about this question.

**Comment by jacobkeeler** [comment]
PR: #1513

**Comment by LuxoftAKutsan** [comment]
@jacobkeeler @JackLivio I can see merged PR related to this issue. Should it be closed?

**Comment by LuxoftAKutsan** [comment]
Priority is hight
Loss of functionality. Wrong result code returned from SDL

**Comment by JackLivio** [comment]
Fixed by #1513 and #1486
